layouts/partials/search-input.html

```diff
@@ -22,11 +22,20 @@
   >
 </div>
 {{ else if .Site.Params.k8s_search }}
+
+{{ $lang := .Site.Language.Lang }}
+{{ $searchFile := printf "content/%s/search.md" $lang }}
+
 <div class="search-bar">
   <i class="search-icon fas fa-search"></i>
   <input
     type="search"
     name="q"
+    {{ if fileExists $searchFile }}
+      data-search-page="{{ "search/" | relLangURL  }}"
+    {{ else }}
+      data-search-page="{{ "search/" | absURL }}"
```

Comment:
This is a big fragile; a better way is to find the site-relative URL of the search page for [English | the primary localization].
But, it'll work for now.

Reply:
Ok, I've made it a relative url so that it will work on local instances.